docs/source/essentials/errors.mdx

```diff
@@ -10,26 +10,21 @@ Whenever you execute a GraphQL operation with Apollo Kotlin (or any other GraphQ
 
 ## Network errors
 
-When calling `execute()`, network errors throw an `ApolloException`. To handle them, wrap your query in a `try/catch` block:
+Network errors are surfaced in the `ApolloResponse.exception` field.
 
 ```kotlin
-try {
-  val response = apolloClient.query(query).execute()
-} catch(exception: ApolloException) {
+val response = apolloClient.query(query).execute()
+if (response.exception != null) {
   // handle exception here
 }
 ```
 
-When using `toFlow()` (e.g. with [subscriptions](../essentials/subscription)) and `watch()` (with the [normalized cache](../caching/query-watchers)), network errors are emitted in `ApolloResponse.exception`.
+This is also true when calling `toFlow()` (e.g. with [subscriptions](../essentials/subscription)) and `watch()` (with the [normalized cache](../caching/query-watchers)).
 
 ```kotlin
-apolloClient.query(query).toFlow().collect { response ->
+apolloClient.subscription(subscription).toFlow().collect { response ->
   if (response.exception != null) {
     // handle network errors
-  } else if (response.hasErrors()) {
-    // handle GraphQL errors
-  } else {
-    // no errors, handle data
```

Comment:
Why remove that part?

Reply:
It's just that we talk about GraphQL errors in the next section. It's spoiling it! :)

Reply:
Fair enough. Ultimately, we'll want to be opinionated about how to handle errors and use the same pattern everywhere for SEO purposes...